bmaptools/BmapHelpers.py

```diff
@@ -83,3 +94,72 @@ def program_is_available(name):
             return True
 
     return False
+
+
+def get_mount_point(filepath):
+    """
+    Return the mount point the file object file resides on.
+    """
```

Comment:
The convention is to have a newline between the docstring and the code. Also for sort comments the quotes can be on the same line.